factory/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""All necessary class initializations."""
+from db.facade import DBFacade
+from db.dynamodb import DynamoDB
+from command.core import Core
+from slackclient import SlackClient
+import os
+
+
+def make_core():
+    """
+    Initialize and returns a :class:`command.core.Core` object.
+
+    Requires that environmental variable ``SLACK_API_TOKEN`` to be set. Please
```

Comment:
`DynamoDb` also requires an environment variable to be read, right? That should be noted here, and we might want to consider making that explicit - as in, pass in the value of that environment variable into the DynamoDb constructor. (Thoughts, anyone?)